ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlanner.java

```diff
@@ -610,4 +629,20 @@ public void requestMonteCarloPlannerReset()
    {
       resetMonteCarloFootstepPlanner = true;
    }
+
+   public void addWayPointToList(Pose3D leftFootGoalPose, Pose3D rightFootGoalPose)
```

Comment:
Not only can we walk to a goal, we can add waypoints which it will walk too in the order they were placed, still requires holding control to walk forward though